samples/samples-csharp/TriggerBindingSamples/PrimaryKeyNotPresentTrigger.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Extensions.Sql.Samples.Common;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Sql.Samples.TriggerBindingSamples
+{
+    public static class PrimaryKeyNotPresentTrigger
```

Comment:
Functions used to test error cases should go in the test/Integration/test-csharp folder.